styles.css

```diff
@@ -371,6 +371,9 @@ form textarea {
   border: none;
 }
 
+#desktop-contactform{
+  display: none;
+}
 @media (min-width: 768px) {
   #navbar-grid {
     display: grid;
```

Comment:
- Hey, great job making the change, however, the list of nav items meant for the desktop version is no longer visible. please, kindly fix this.


![image](https://github.com/AbdulSalamAbro/portfoliowebsite/assets/59397159/ad52dfe8-d1bc-442a-a2fa-35ab7c45fbec)

![image](https://github.com/AbdulSalamAbro/portfoliowebsite/assets/59397159/daf11434-4e7e-4789-87cb-3c52e970a8bf)

Reply:
I just place that code outside the media query for desktop I don't know why it is not displaying now on desktop, I am trying to fix that